pandas/core/frame.py

```diff
@@ -9416,6 +9417,10 @@ def corr(
             Minimum number of observations required per pair of columns
             to have a valid result. Currently only available for Pearson
             and Spearman correlation.
+        calculate_diagonal : bool, optional
```

Comment:
this is descriptive, but is there precedent for this type of naming, e.g numpy?

Reply:
No. I came up with this myself. I'm not familiar with Numpy naming convention. Any suggestion ?